plugins/node.d/icinga

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf
+
+ICINGACLI=${ICINGACLI:-$(which icingacli)}
+JQ=${JQ:-$(which jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ "$ICINGACLI" != ""  -a "$JQ" != "" ] ; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli or jq)"
+        exit 1
+    fi
+fi
+export TERM=vt100
+
+if [ "$1" = "config" ]; then
+
+    echo "multigraph icinga_hosts"
+    echo "graph_title Icinga Host Checks"
+    echo 'graph_args --base 1000'
+    echo 'graph_vlabel Count'
+    echo 'graph_category icinga'
+    echo "up.label Up"
+    echo "down.label Down"
+    echo "unreachable.label Unreachable"
+    echo "pending.label Pending"
+    echo "up.draw AREA"    
+    echo "down.draw STACK"
+    echo "unreachable.draw STACK"
+    echo "pending.draw STACK"
+
+
+    echo "multigraph icinga_services"
+    echo "graph_title Icinga Service Checks"
+    echo 'graph_args --base 1000'
+    echo 'graph_vlabel Count'
+    echo 'graph_category icinga'
+    echo "ok.label Ok"
+    echo "warning.label Warning"
+    echo "critical.label Critical"
+    echo "unknown.label Unknown"
+    echo "pending.label Pending"
+    echo "ok.draw AREA"
+    echo "warning.draw STACK"
+    echo "critical.draw STACK"
+    echo "unknown.draw STACK"
+    echo "pending.draw STACK"
+
+    exit
+fi
+
+output=$($ICINGACLI monitoring list hosts --format=json)
```

Comment:
Please quote variables (here: `$ICINGACLI`) in order to avoid whitespace issues.
`shellcheck` is your friend! :)

Reply:
this is also done